SabreTools.FileTypes/BaseFile.cs

```diff
@@ -62,6 +62,11 @@ public class BaseFile
         /// </summary>
         public string Parent { get; set; }
 
+        /// <summary>
+        /// Childs of the file
+        /// </summary>
+        public string Child { get; set; }
```

Comment:
"Children" not "Child". You probably also want a list or array of children names, not just a single `string`.

Reply:
ohh, yep :(